src/tools/tidy/src/deps.rs

```diff
@@ -98,13 +98,14 @@ const EXCEPTIONS: ExceptionList = &[
     ("ryu", "Apache-2.0 OR BSL-1.0"), // BSL is not acceptble, but we use it under Apache-2.0                       // cargo/... (because of serde)
     ("self_cell", "Apache-2.0"),                             // rustc (fluent translations)
     ("snap", "BSD-3-Clause"),                                // rustc
-    ("wasm-encoder", "Apache-2.0 WITH LLVM-exception"),      // rustc
-    ("wasm-metadata", "Apache-2.0 WITH LLVM-exception"),     // rustc
-    ("wasmparser", "Apache-2.0 WITH LLVM-exception"),        // rustc
-    ("wast", "Apache-2.0 WITH LLVM-exception"),              // rustc
-    ("wat", "Apache-2.0 WITH LLVM-exception"),               // rustc
-    ("wit-component", "Apache-2.0 WITH LLVM-exception"),     // rustc
-    ("wit-parser", "Apache-2.0 WITH LLVM-exception"),        // rustc
+    ("wasi-preview1-component-adapter-provider", "Apache-2.0 WITH LLVM-exception"), // rustc
+    // ("wasm-encoder", "Apache-2.0 WITH LLVM-exception"),      // rustc
+    // ("wasm-metadata", "Apache-2.0 WITH LLVM-exception"),     // rustc
+    // ("wasmparser", "Apache-2.0 WITH LLVM-exception"),        // rustc
+    // ("wast", "Apache-2.0 WITH LLVM-exception"),              // rustc
+    // ("wat", "Apache-2.0 WITH LLVM-exception"),               // rustc
+    // ("wit-component", "Apache-2.0 WITH LLVM-exception"),     // rustc
+    // ("wit-parser", "Apache-2.0 WITH LLVM-exception"),        // rustc
```

Comment:
`wasmparser` changed from `Apache-2.0 WITH LLVM-exception` to `Apache-2.0 WITH LLVM-exception OR Apache-2.0 OR MIT`, but we happen to have both versions (before and after license change) in the lockfile. Regardless of what I do here it seems like tidy complains about one or the other.

This will be resolved if `object` can update their dependency https://github.com/gimli-rs/object/pull/715

Reply:
There was a release to `object`, this is resolved.